app/core/store/modules/courses.coffee

```diff
@@ -1,25 +1,32 @@
+Vue = require('vue').default
```

Comment:
Can remove this since `Vue` is globally available

Reply:
done